src/app/progress/PursuitItem.m.scss

```diff
@@ -13,14 +13,14 @@
   height: calc(var(--item-size));
   box-sizing: border-box;
   border: 1px solid #666;
-  background-size: cover;
+  background-size: calc(var(--item-size) + 2px) calc(var(--item-size) + 2px);
+  background-position: center;
```

Comment:
I like the way this looks, but the only weird part is it ends up stretching the image (with the recommended 50px image size). I'm generally trying to get everything to line up so at 50px there's a 2 to 1 ratio of source to screen pixels.